homework/index.js

```diff
@@ -28,9 +30,37 @@
     });
     return elem;
   }
-
   function renderRepoDetails(repo, ul) {
-    createAndAppend('li', ul, { text: repo.name });
+    // eslint-disable-next-line prefer-template
+    const li = createAndAppend('li', ul, { id: 'item' });
+    const repoName = createAndAppend('p', li);
+    repoName.innerHTML = "<span class = 'nameItem'>Repository:</span> ";
```

Comment:
Using `.innerHTML` to create HTML element this way is inconsistent with the rest of the code that uses `createAndAppend()`. Please use only `createAndAppend()` to create elements in this homework.

Reply:
I was looking for a way to make change the font to be bold, but now i don't need it any more